.vib/spring-cloud-dataflow/goss/spring-cloud-dataflow.yaml

```diff
@@ -0,0 +1,16 @@
+command:
+  check-spring-cloud-dataflow:
+    # As the version is shown in the output we check it here
+    timeout: 20000
+    exec: timeout --preserve-status 15 java -jar /opt/bitnami/spring-cloud-dataflow/spring-cloud-dataflow.jar
```

Comment:
Looks like the timeout of 15 seconds may be too close to its limit.
In my case, it took 12 seconds to reach the 'Started DataflowServerApplication':
```
2023-05-24 08:44:53.598  INFO 7 --- [           main] o.s.c.d.s.s.DataFlowServerApplication    : Started DataFlowServerApplication in 11.838 seconds (JVM running for 12.597)
```

I think we should increase this to at least 30 with a goss timeout of 40000, considering executions in CP could take a bit longer than locally.